### DISCUSSION TITLE
Draft v1.2.0 release

### DISCUSSION THREAD

**Discussion by godber** [terascope/standard-assets#944] [Announcements]
https://github.com/terascope/standard-assets/releases/tag/untagged-0c943930ac990a497122

**Comment by godber** [comment]
Make sure the version is bumped in package.json and assets.json and release.

We're playing with the idea of using discussions to coordinate releases.  It may not make sense, just an Nth place to put stuff that gets ignored.

**Comment by busma13** [comment]
@sotojn this draft is ready for you to review.

**Comment by busma13** [comment]
it looks like the hash in the link to the draft changes on every draft save. I guess we just need to remember to edit the link every time.

**Comment by sotojn** [comment]
I have finalized my review and made a `pre-release` of `v1.2.0`

https://github.com/terascope/standard-assets/releases/tag/v1.2.0

**Reply by sotojn** [reply]
Here was the final checklist that should all be validated and posted before a release is made:

- [x] Ensure `asset.json`, `asset/package.json`, and `package.json` are in sync and on correct version for release
- [x] Check for any unfinished PR's that could make it into the next release
- [x] Check the draft for any other contributions to the master branch since last review of draft. (Auto generate new release notes)
- [x] Ensure release-notes are refactored to fit the context of the end user
- [x] After all the above is checked, request peer review before release 

Here is a template for easy copy and paste. 

```
- [ ] Ensure `asset.json`, `asset/package.json`, and `package.json` are in sync and on correct version for release
- [ ] Check for any unfinished PR's that could make it into the next release
- [ ] Check the draft for any other contributions to the master branch since last review of draft. (Auto generate new release notes)
- [ ] Ensure release-notes are refactored to fit the context of the end user
- [ ] After all the above is checked, request peer review before release 
```

**Comment by sotojn** [comment]
The release workflow ran successfully.

https://github.com/terascope/standard-assets/actions/runs/11858775075
